specification/eventgrid/Azure.Messaging.EventGrid/main.tsp

```diff
@@ -100,63 +103,96 @@ namespace EventGrid {
+    ReleaseEventIn60Seconds: 60,
+
+    @doc("Release the event after 600 seconds.")
+    ReleaseEventIn600Seconds: 600,
+
+    @doc("Release the event after 3600 seconds.")
+    ReleaseEventIn3600Seconds: 3600,
+  }
+
   @doc("The result of the Publish operation.")
   model PublishResult {}
 
   @doc("The result of the Acknowledge operation.")
   model AcknowledgeResult {
-    @doc("Array of LockToken values for failed cloud events. Each LockToken includes the lock token value along with the related error information (namely, the error code and description).")
+    @doc("Array of FailedLockToken for failed cloud events. Each FailedLockToken includes the lock token along with the related error information (namely, the error code and description).")
     failedLockTokens: FailedLockToken[];
 
-    @doc("Array of lock tokens values for the successfully acknowledged cloud events.")
+    @doc("Array of lock tokens for the successfully acknowledged cloud events.")
     succeededLockTokens: string[];
   }
 
   @doc("The result of the Release operation.")
   model ReleaseResult {
-    @doc("Array of LockToken values for failed cloud events. Each LockToken includes the lock token value along with the related error information (namely, the error code and description).")
+    @doc("Array of FailedLockToken for failed cloud events. Each FailedLockToken includes the lock token along with the related error information (namely, the error code and description).")
     failedLockTokens: FailedLockToken[];
 
-    @doc("Array of lock tokens values for the successfully released cloud events.")
+    @doc("Array of lock tokens for the successfully released cloud events.")
     succeededLockTokens: string[];
   }
 
   @doc("The result of the Reject operation.")
   model RejectResult {
-    @doc("Array of LockToken values for failed cloud events. Each LockToken includes the lock token value along with the related error information (namely, the error code and description).")
+    @doc("Array of FailedLockToken for failed cloud events. Each FailedLockToken includes the lock token along with the related error information (namely, the error code and description).")
     failedLockTokens: FailedLockToken[];
 
-    @doc("Array of lock tokens values for the successfully rejected cloud events.")
+    @doc("Array of lock tokens for the successfully rejected cloud events.")
     succeededLockTokens: string[];
   }
 
-  @doc("Array of lock token strings for the corresponding received Cloud Events to be released.")
+  @doc("The result of the RenewLock operation.")
+  model RenewCloudEventLocksResult {
+    @doc("Array of FailedLockToken for failed cloud events. Each FailedLockToken includes the lock token along with the related error information (namely, the error code and description).")
+    failedLockTokens: FailedLockToken[];
+
+    @doc("Array of lock tokens for the successfully renewed locks.")
+    succeededLockTokens: string[];
```

Comment:
It's a little confusing to see the property names both end in "Tokens" but one is a model where the other is a string (per item). What does the string represent? Is it a token name or ID? If you keep a string here - not very flexible, mind you - you should add "Names" or "Ids" as the suffix depending on what it represents e.g., `succeededLockTokenIds`.

Reply:
As discussed, we are keeping the existing "FailedLockToken" model